src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,479 @@
+<div id="file-preview-card">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-preview-card can be used to display file previews. A css grid layout is used to define the areas in which certain elements can be placed.
+    </p>
+
+    <h4>Center</h4>
+    <p>
+        Apply the
+        <span class="highlight">
+            file-preview-card__center
+        </span>
+        ${" "}class to set the content to be displayed in the card, which will be centered.${" "}
+    </p>
+    <p>
+        If the content is an
+        <span class="highlight">
+            img
+        </span>
+         element then it will automatically fit the card.
+    </p>
+    <p>
+        Apply the
+        <span class="highlight">
+            progress-bar
+        </span>
+         class to a child progress element.
+    </p>
+    <p>
+        Apply the
+        <span class="highlight">
+            inline-notice--attention
+        </span>
+         class to a child attention inline notice element.
+    </p>
+
+    <h4>Menu Action</h4>
+    <p>
+        Apply the
+        <span class="highlight">
+            file-preview-card__menu-action
+        </span>
+        ${" "}class to place an action button in the top right corner of the card.
+    </p>
+
+    <h4>Label</h4>
+    <p>
+        Apply the
+        <span class="highlight">
+            file-preview-card__label
+        </span>
+        ${" "}class to place a label in the bottom left corner of the card. This label can also be used a CTA button.
+    </p>
+
+    <h4>Center Action</h4>
+    <p>
+        Apply the
+        <span class="highlight">
+            file-preview-card__center-action
+        </span>
+        ${" "}class to place an action button in the center of the card.
+    </p>
+
+    <h3>Uploading</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__center">
+
+                        <progress class="progress-bar" value="60" max="100"/>
+                    </div>
+                    <div class="file-preview-card__menu-action">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Cancel upload"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-close-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <div class="card">
```

Comment:
Is this `"card"` class used anymore? Also, do we need this extra `div` wrapping everything?

Reply:
good catch, removed!